doc/sphinx/Pacemaker_Remote/kvm-tutorial.rst

```diff
@@ -266,108 +65,31 @@ Just be sure to configure the guest with a hostname and a static IP address
 Configure Firewall on Guest
 ___________________________
 
-On each guest, allow cluster-related services through the local firewall,
-following the same procedure as in `Configure Firewall on Host`_.
+On each guest, allow cluster-related services through the local firewall.
 
 Verify Connectivity
 ___________________
 
 At this point, you should be able to ping and ssh into guests from hosts, and
 vice versa.
 
-Configure pacemaker_remoted
-___________________________
+Configure pacemaker_remote on Guest Node
+________________________________________
 
-Install ``pacemaker_remoted``, and enable it to run at start-up. Here, we also
-install the ``pacemaker`` package; it is not required, but it contains the
-dummy resource agent that we will use later for testing.
+Install the pacemaker_remote daemon on the guest node.
 
 .. code-block:: none
 
-    # yum install -y pacemaker pacemaker-remote resource-agents
-    # systemctl enable pacemaker_remote.service
+    # yum install -y pacemaker-remote resource-agents pcs
 
-Copy the authentication key from a host:
+Additionally, install ``pacemaker_remoted``. Here, we also
+install the ``pacemaker`` package; it is not required, but
+it contains the dummy resource agent that we will use later
+for testing.
```

Comment:
pacemaker-remote was already installed with the previous command. Really, we can merge the two yum install commands into one.